blackboxopt/optimizers/testing.py

```diff
@@ -321,7 +372,8 @@ def handles_conditional_space(
 
 ALL_REFERENCE_TESTS = [
     optimize_single_parameter_sequentially_for_n_max_evaluations,
-    is_deterministic_with_fixed_seed,
+    is_deterministic_with_fixed_seed_and_mixed_space,
+    is_deterministic_with_fixed_seed_and_multiple_evaluations,
```

Comment:
Is it the presence of multiple parameter types or in general a higher search space dimensionality that makes the difference here? If it's the dimensionality independent of the types, let's rename the reference test from `_and_mixed_space` to something along the lines of `_and_large_space` or `_and_4D_space` (or what ever the dimensionality is). Otherwise, seeing the two tests side by side, it might be misread as the parameter types making the difference. Whereas if that's indeed what makes the difference, everything is alright :)

Reply:
Actually, I don't know what makes the difference. I'll give it a try and rename accordingly.

Reply:
Renamed